packages/hooks/src/createRunHook.js

```diff
@@ -21,21 +21,12 @@ function createRunHook( hooks, returnFirstArg ) {
 	 * @return {*}               Return value of runner, if applicable.
 	 */
 	return function runHooks( hookName, ...args ) {
-
-		if ( ! validateHookName( hookName ) ) {
```

Comment:
This removes all validation from hooks names when applying them.

Reply:
>This removes all validation from hooks names when applying them.

What's the worst case scenario here?

The assumption I'd made, noted in extended commit description of a0e6eb6, is that validation of the hooks takes place when they're added, and if no hooks exist which match the hook name (a given if it's an invalid hook name), nothing would occur.

Reply:
Seems reasonable 👍